tests/test_units/test_rms_context.py

```diff
@@ -554,6 +578,32 @@ def test_gridproperty_export_file_set_name(rmssetup, rmsglobalconfig, gridproper
     )
 
 
+@inside_rms
+def test_gridproperty_export_with_geometry(
+    rmssetup, rmsglobalconfig, grid, gridproperty
+):
+    """Export the the grid and the gridprop(s) with connected geometry."""
+    logger.info("Active folder is %s", rmssetup)
+    os.chdir(rmssetup)
+
+    edata = dataio.ExportData(
+        config=rmsglobalconfig, content="depth"
```

Comment:
Should the content be more granular then "depth"? Perhaps add `grid_geometry` as a content, not sure. But `depth` feels wrong. (`data.vertical_domain` defines if this is depth or time.)